src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1708,71 +1708,109 @@ Below is an example for adding a certificate to a Java cacerts file:
 
-===== Atomic Clauses
+    GET http://localhost:8080/catalog/buckets/{bucket_id}/resources/{name}
 
-    name="My First Workflow"
-    project_name="Templates"
+NOTE: By default, the catalog will always return the latest version of the workflows.
 
-===== Key/Value Clauses
+=== Workflow versioning
 
-    generic_information("Infrastructure", "Amazon EC2")
-    variable("CPU", "4")
+The Catalog provides versioning of workflows. You can list all the revisions of a particular Workflow using the HTTP request:
 
-===== Expressions and Wildcard
+    GET http://localhost:8080/catalog/buckets/{bucketId}/resources/{name}/revisions
 
-Let the following Workflows:
-[width="100%",frame="topbot",options="header", cols="30%<"]
-|==============================================================================================
-| Name                         | Project Name | GI Name | GI Value | Variable Name     | Variable Value
-| _My First Workflow_          | Tutorials    |         |          |                   |
-| _Create OpenStack Instance_  | Deployment   | Infrastructure | OpenStack  | Core     | 4
-| _Stop OpenStack Instance_    | Lifecycle    | Infrastructure | OpenStack  |          |
-| _New Amazon Instance_        | Deployment   | Infrastructure | Amazon EC2 | Nb_CPU   | 2
-|==============================================================================================
+The revision of workflow is identified by `commit_time`. Commit time specifies the time when the workflow's version was added in the catalog.
+You can fetch a Workflow detailed metadata information using its `name` and `commit_time`:
 
-They are matched against these query expressions:
-[width="100%",frame="topbot",options="header"]
-|=================================================================================================================================
-| Expression                                          | Matches
-| name = "My First Workflow"                          | _My First Workflow_
-| name = "My*"                                        | _My First Workflow_
-| project_name = "\*ploy*"                            |_Create OpenStack Instance_ and _New Amazon Instance 1_
-| project_name = "\*ploy*" AND variable("\*CPU", "*") |_New Amazon Instance 1_
-|=================================================================================================================================
+    GET http://localhost:8080/catalog/buckets/{bucketId}/resources/{name}/revisions/{commit_time}
+
+A Workflow Revision is created from a ProActive XML <<_glossary_workflow,*Workflow*>> definition.
+Beside from the _id_, _name_ and _project_name_ of the Workflow, a few more information are indexed from the XML content such as _<<_generic_information>>_ and _<<_workflow_variables>>_.
+
+More detailed information on how to create or modify Workflows and Buckets are available in the link:../admin/ProActiveAdminGuide.html#_catalog[Admin guide].
+
+=== Catalog security and groups management
```

Comment:
The *Catalog* is secured with a sessionId. Every request to the *Catalog* must have a valid sessionId inside the request header. 
The security mechanism is based on groups, the group GROUP:public-objects can be access by any user with a valid sessionId. Other groups can be accessed (read and write) if the sessionId corresponds to a user which is part of that group.
As an example, a user which is part of the interns group (GROUP:interns) can see and change buckets and their contents which belong to the GROUP:interns. 
A user which gets a list of buckets can only see the buckets to which the user belongs plus the GROUP:public-objects associated buckets. 

The sessionId is validated against the Scheduler REST api, so if the *Catalog* does not have access (network outage, firewall) to the Scheduler, no access to the *Catalog* is possible.